mtest/testutils.cpp

```diff
@@ -161,6 +161,7 @@ bool MTest::compareFilesFromPaths(const QString& f1, const QString& f2)
       {
       QString cmd = "diff";
       QStringList args;
+      args.append("-w");  // ignore any whitespace differences
```

Comment:
Can you please update the `qDebug()` in line 170 to show the full and real command line?

Reply:
Funny, i was actually thinking about the same, but as `--strip-trailing-cr` also was missing there i thought i'd leave it there, but of course you're right in the sense that this is helpful improvement for debugging purposes.

As this is an enhancement in itself, added as a separate commit → 7ebdd4688

Reply:
Just amended that last commit, it is now as [DRY](https://de.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself) as it should be.

See →e0ddd738c

Reply:
Also took the liberty to remove some commented debug codelines in c5ea88f88 as they didn't seem to provide added value. They were commented out 2 - 5 years ago, should anybody urgently need them, they are still in the history. This way the code is more readable.

Reply:
Unfortunately whitespace is sometimes significant in XML (at least how MuseScore uses it) so I don't think it is safe to ignore it.


![image](https://user-images.githubusercontent.com/11011881/79938240-f83b2880-8453-11ea-8c72-6c5488be9b2d.png)



```xml
          <StaffText>
            <text>This text
spans
    multiple
lines.
</text>
            </StaffText>
```